pinot-core/src/main/java/org/apache/pinot/core/data/manager/realtime/LLRealtimeSegmentDataManager.java

```diff
@@ -1437,6 +1438,7 @@ public LLRealtimeSegmentDataManager(SegmentZKMetadata segmentZKMetadata, TableCo
           new SegmentErrorInfo(now(), "Failed to initialize the StreamMessageDecoder", e));
       throw e;
     }
+    _clientId = streamTopic + "-" + _partitionGroupId;
```

Comment:
Client Id was `_clientId = _tableNameWithType + "-" + streamTopic + "-" + _partitionGroupId;` before this PR.

We should still use that right? We didn't want to change the clientId afaik and only wanted the metricKeyName to be set separately.

Reply:
+1